providers/shared/components/ddb/id.ftl

```diff
@@ -0,0 +1,311 @@
+[#ftl]
+
+[@addComponent
+    type=DDB_COMPONENT_TYPE
+    properties=
+        [
+            {
+                "Type"  : "Description",
+                "Value" : "A managed NoSQL database instance"
+            },
+            {
+                "Type" : "Note",
+                "Value" : "Minor Version Upgrades - Are presently ignored by DocDB",
+                "Severity" : "warning"
+            }
```

Comment:
If we aren't using minor version support it should be removed

Reply:
Done